internal/plugincommon/validations.go

```diff
@@ -15,3 +16,40 @@ func ValidateFChain(fChain map[cciptypes.ChainSelector]int) error {
 	}
 	return nil
 }
+
+var ErrValidationError = errors.New("report validation errors")
+var ErrInvalidReport = errors.New("invalid report")
```

Comment:
Two things: naming nit and godoc
```suggestion
// ErrReportValidation is used when the validation being performed in order to validate a report has errored.
// This does not necessarily mean that the report is valid or invalid.
var ErrReportValidation = errors.New("report validation error")
// ErrInvalidReport is used when a report is found to be invalid.
var ErrInvalidReport = errors.New("invalid report")
```